src/AdvanceSteelNodes/Util/Selection.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AdvanceSteel.Nodes.Plates;
+using Autodesk.AdvanceSteel.CADAccess;
+using Autodesk.DesignScript.Runtime;
+using DynGeometry = Autodesk.DesignScript.Geometry;
+using SteelGeometry = Autodesk.AdvanceSteel.Geometry;
+using Autodesk.AdvanceSteel.DotNetRoots.DatabaseAccess;
+using SteelServices = Dynamo.Applications.AdvanceSteel.Services;
+using Autodesk.AdvanceSteel.ConstructionTypes;
+using System.Collections.Generic;
+using Autodesk.AdvanceSteel.Geometry;
+using System.Linq;
+using System;
+
+
+namespace AdvanceSteel.Nodes.Selection
+{
+	/// <summary>
+	/// Store Bolts properties in a Node to pass to Bolt Node
+	/// </summary>
+	public class ObjectSelection
+	{
+		internal ObjectSelection()
```

Comment:
this class looks like a static class, it should be "public static class ObjectSelection". We should remove the internal default constructor